### PR TITLE
clean up git stuff

### PR DESCRIPTION
**Pull request by johnduffell**
## Why are you doing this?

Just a few things to clean up  See inline comments.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-If this PR is a fix, please include a link to the original PR that introduced
-the breakage for reference.
+This pr adds a widget to the doogle so that we can buy a sub from any page in one click
+For detailed changes see the inline self-review comments.
 -->
 
-[**Trello Card**](https://trello.com)
-
-## Changes
+## Why are you doing this?
```
Comment (johnduffell) [inline .github/PULL_REQUEST_TEMPLATE.md:9]: I did a quick audit of recent PRs (other than mine) and only one of them actually wrote the WHY, and usually the WHAT was written under the WHY title.  The WHY was usually not included at all.
So it makes sense to make the top title WHAT, and put the WHY below as an optional section.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
-* Change 2
+<!--
+Remember, PRs are documentation for future contributors.
+-->
 
-## Did you enjoy your PR experience?
+## Useful links (trello, google docs, previous PRs, etc)
 
- - [ ] [PR experience rated](https://forms.gle/N6FsTGG8JQFGV4Ha9)
+<!-- trello -->
```
Comment (johnduffell) [inline .github/PULL_REQUEST_TEMPLATE.md:17]: the existing trello link is finickity and it's easier to just paste the link as it comes

File: build.sbt [hunk 3]
```diff
@@ -31,7 +31,7 @@ lazy val release = Seq[ReleaseStep](
 
 inThisBuild(Seq(
   organization := "com.gu",
-  scalaVersion := "2.12.10",
+  scalaVersion := "2.12.12",
```
Comment (johnduffell) [inline build.sbt:34]: everything still seems to build fine.  We should get onto 2.13 before scala 3 is released as the 2.13->3 migration can be automated, also the libraryDependencies are cross compatible 2.13<->3.
I think the main thing blocking us here was internal library dependencies e.g. event producer.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (johnduffell) [reply .github/PULL_REQUEST_TEMPLATE.md:17]: you like it to be specifically a [....](...) style link, or you just want it to be prominent and specific?

It was the [...](...) that always caused me hassle - but I just thought, I always carefully paste the link inside the (....) but there's nothing stopping me personally pasting over it completely every time!

File: build.sbt [hunk 4]
```diff
@@ -89,8 +89,6 @@ lazy val root = (project in file("."))
 
 lazy val testScalastyle = taskKey[Unit]("testScalastyle")
 
-lazy val setupGitHook = taskKey[Unit]("Set up a pre-push git hook to run the integration tests")
```
[the remark this answers is not in the document]
Comment (TomPretty) [reply build.sbt:92]: I don't think I've ever intentionally *not* used `--no-verify`. Very happy to see this gone.

**Review by TomPretty — APPROVED** [review]
Looks great to me 👍

**Pull request merged**